core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_sidebar_directive.html

```diff
@@ -112,6 +112,9 @@
   simple-editor-sidebar .highlighted {
     background-color: #c5e1a5;
   }
+  simple-editor-sidebar .error {
```

Comment:
Then please do so. That is the first thing one should do when trying to figure out whether something exists in the code or not, or where some functionality is. Only if you can't find it this way, should you ask (and also explain what you've tried).

In case it helps, you can use a bash invocation similar to: `grep "thing to search for" . -r --exclude-dir=third_party`. The 'thing to search for' here could be 'material', for example.

Re your question to @andromfins, I think that this PR should match the error-handling part of the mocks and should implement whatever is needed to do so, including any necessary icons.